parm/archive/gdas_restarta.yaml.j2

```
@@ -47,7 +47,7 @@ gdas_restarta:
 
         # Snow configuration yaml
         {% if DO_JEDISNOWDA %}
-        - "{{ COMIN_CONF | relpath(ROTDIR) }}/{{ head }}letkfoi.yaml"
+        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowanl.yaml"
```

```suggestion
        - "{{ COMIN_SNOW_ANALYSIS | relpath(ROTDIR) }}/{{ head }}snowanlvar.yaml"
```
Keep consistent to the defined `expected_keys = ['snowanlvar']` in `ush/python/pygfs/task/snow_analysis.py`